pytorch_lightning/accelerators/accelerator.py

```diff
@@ -44,15 +45,29 @@ class Accelerator:
     One to handle differences from the training routine and one to handle different precisions.
     """
 
-    def __init__(self, precision_plugin: PrecisionPlugin, training_type_plugin: TrainingTypePlugin) -> None:
+    def __init__(
+        self, training_type_plugin: TrainingTypePlugin, precision_plugin: Optional[PrecisionPlugin] = None
+    ) -> None:
         """
         Args:
-            precision_plugin: the plugin to handle precision-specific parts
             training_type_plugin: the plugin to handle different training routines
+            precision_plugin: the plugin to handle precision-specific parts
+
+        Notes:
+            precision_plugin is deprecated and will be removed soon
+            User `training_type_plugin(precision_plugin)` instead
         """
-        self.precision_plugin = precision_plugin
+
         self.training_type_plugin = training_type_plugin
 
+        if precision_plugin:
+            """
+            .. deprecated
+                precision_plugin parameter is deprecated will be removed soon.
+                Use :`training_type_plugin(precision_plugin) instead.
+            """
```

Comment:
nit: Triple quotes should be for docstrings, which this is not. Can you use comments instead?

Reply:
Comments for deprecation notes?

Reply:
This must have been a merge resolution issue because I had already fixed this, including the formatting @four4fish.

Reply:
Yep, that's mostly it, but you can keep the text ".. deprecated soon" instead of v1.7 as the target :)

Reply:
is 
```
 .. deprecated:: soon
                    The ``precision_plugin`` parameter has been deprecated and will be removed in soon
                    Pass the precision plugin as a parameter to the ``TrainingTypePlugin`` instead.
```
A valid format? Or do you think use comments better?

Reply:
Nope :) 

This is our standard format: 

```
.. deprecated:: v1.6
                    The ``precision_plugin`` parameter has been deprecated in v1.6 and will be removed soon
                    Pass the precision plugin as a parameter to the ``TrainingTypePlugin`` instead.
```

Reply:
How about we just use comments then? we gonna remove before 1.6, don't want to misleading user

Reply:
Comments don't show in the documentation. We dont expect them to read the source files, right?
We could just drop the version number

.. deprecated::
    The ``precision_plugin`` parameter has been deprecated and will be removed soon
    Pass the precision plugin as a parameter to the ``TrainingTypePlugin`` instead.